distributed/deploy/tests/test_runner.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+from contextlib import suppress
+from time import time
+
+import pytest
+
+from dask.distributed import Client
+
+from distributed.deploy.runner import AsyncCommWorld, AsyncRunner
+from distributed.utils_test import loop  # noqa: F401
+
+
+@pytest.mark.asyncio
+@pytest.mark.timeout(10)
+async def test_runner(loop):
+    commworld = AsyncCommWorld()
+
+    async def run_code(commworld):
+        with suppress(SystemExit):
```

Comment:
This seems atypical in Dask tests.  If we go with some sort of dask-runner or dask-server class I think that we should reconsider explicitly calling `sys.exit(0)` when that class finishes up.